services/quests/docs/qdl_guide.md

```diff
@@ -501,6 +503,17 @@ _Another card_ (#another)
 After clicking "Jump me!", the app will immediately proceed to this card.
 ```
 
+## Formatting
```

Comment:
Nice. Thoughts on adding something like this to the template/starter quest when you first load the quest creator?